test/helpers/kubectl.ts

```diff
@@ -1,21 +1,29 @@
 import { exec } from 'child-process-promise';
-import { chmodSync, writeFileSync, existsSync } from 'fs';
+import { chmodSync, writeFileSync, existsSync, unlinkSync } from 'fs';
 import { platform } from 'os';
 import { resolve } from 'path';
 import * as needle from 'needle';
 import * as sleep from 'sleep-promise';
 
-export async function downloadKubectl(): Promise<void> {
+/**
+ * @param version For example: "v1.18.0"
+ */
+export async function downloadKubectl(version?: string): Promise<void> {
   const kubectlPath = resolve(process.cwd(), 'kubectl');
   if (existsSync(kubectlPath)) {
-    return;
+    if (version === undefined) {
+      return;
+    }
+
+    // Always start clean when requesting a specific version.
+    unlinkSync(kubectlPath);
   }
 
-  console.log('Downloading kubectl...');
+  console.log(`Downloading kubectl ${version || ''}...`);
```

Comment:
not sure about it, it looks like code written for backwards compatibility instead of a function written in the first place to solve the problem of "download Kubectl"